### ISSUE TITLE
regarding to val.txt format

### ISSUE THREAD

**Issue by djsg** [kratzert/finetune_alexnet_with_tensorflow#75]
Hi,

you only explained the format for train.txt file. I guess the class label refers to different object: for example, if I have only two objects cat and dog, I can use 1 (cat) and 2 (dog) as class label. Am I right?

For test images, is it the same format? If so, why do we need class label here? We are supposed to let AlexNet to guess what object is in that image, aren't we?

**Comment by kratzert** [comment]
It's the same format of course. You want labels for the validation set, since it is used to validate your trained network on other but the training data. You don't need labels for the test set
